app/scripts/controllers/provider-approval.js

```diff
@@ -12,7 +12,7 @@ class ProviderApprovalController extends SafeEventEmitter {
    *
    * @param {Object} [config] - Options to configure controller
    */
-  constructor ({ closePopup, initState, keyringController, openPopup, preferencesController } = {}) {
+  constructor ({ closePopup, initState = {}, keyringController, openPopup, preferencesController } = {}) {
```

Comment:
Nit: This change isn't needed anymore

Not that it looks harmful, but it'd avoid a merge conflict with `LoginPerSite` at least.

Reply:
Removed in e41da1823